### ISSUE TITLE
Connecting a split to another split blows up

### ISSUE THREAD

**Issue by ericnewcomer** [nyaruka/floweditor#195]
This will get into what looks like some kind of dispatch loop.

**Comment by ycleptkellan** [comment]
this looks to be specific to `GroupsRouter`; `this.props.nodeToEdit` is `null` for some reason...looking into it

**Comment by ericnewcomer** [comment]
I'm seeing it with non group splits: http://take.ms/HsKe7

```
floweditor.js:1 Uncaught TypeError: Cannot read property 'offsetLeft' of null
    at e.jsPlumbInstance.getOffset (VM296 floweditor.js:1)
    at Object.updateOffsets (VM296 floweditor.js:1)
    at VM296 floweditor.js:1
    at pe (VM296 floweditor.js:1)
    at e.jsPlumbInstance.revalidate (VM296 floweditor.js:1)
    at VM296 floweditor.js:1```

**Comment by ericnewcomer** [comment]
Fixed in #199
